test/suites/alpha/integration/ami_test.go

```diff
@@ -396,7 +396,7 @@ var _ = Describe("AMI", func() {
 			})
 
 			env.ExpectCreated(pod, provider, provisioner)
-			env.EventuallyExpectHealthyWithTimeout(time.Minute*15, pod) // Wait 15 minutes because Windows nodes/containers take longer to spin up
+			env.EventuallyExpectHealthyWithTimeout(time.Minute*20, pod) // Wait 20 minutes because Windows nodes/containers take longer to spin up
```

Comment:
Does the 5 minutes really make a difference here? I was under the impression that 15 minutes was sufficient.